setup.py

```diff
@@ -5,23 +5,23 @@
     readme = f.read()
 
 setup(
-    name             = 'heudiconv',
+    name             = 'pl-heudiconv',
     version          = '0.1',
     description      = 'An app to ...',
```

Comment:
Should we populate this line?

Reply:
I guess it could be populated the same as the README since there's a corresponding sentence there (I think they both came from the project generator). I don't think that this will be exposed anywhere though, so I wouldn't worry about it being perfect. My understanding is that it's used by package registries and tools for giving info on the package. This package won't actually exist on a registry though...it'll just be part of the container image.